exercises/java/src/main/java/com/adaptionsoft/games/uglytrivia/Game.java

```diff
@@ -4,20 +4,26 @@
 import java.util.LinkedList;
 
 public class Game {
-    ArrayList players = new ArrayList();
-    int[] places = new int[6];
-    int[] purses  = new int[6];
+	public static final String POP_CATEGORY = "Pop";
+	public static final String SCIENCE_CATEGORY = "Science";
+	public static final String SPORTS_CATEGORY = "Sports";
+	public static final String ROCK_CATEGORY = "Rock";
+	public static final int WINNING_SCORE = 6;

```

Comment:
I like this being introduced as a constant. This is something I never did. What are your thoughts about having a way to configure the game settings from outside of the class? for example, maybe another person wants to start a game with different rules and wants the winning score to be higher. It would be nice to see how you would go about adding in settings

Reply:
I was thinking about this and potentially implementing a Deck similar to what you had looked at. Then game would basically have a deck and if their were variations of the game with different questions it wouldn't actually matter to the game.

Reply:
Yeah and you can just swap them out without changing anything